proto/osmosis/concentrated-liquidity/query.proto

```diff
@@ -208,7 +208,8 @@ message ClaimableIncentivesResponse {
     (gogoproto.moretags) = "yaml:\"claimable_incentives\"",
     (gogoproto.nullable) = false
   ];
-  repeated cosmos.base.v1beta1.Coin forfeited_incentives = 2 [
+  repeated cosmos.base.v1beta1.DecCoin forfeited_incentives = 2 [
```

Comment:
What is the benefit of returning `DecCoin`? I'm supportive of adding the missed tests but I think returning `Coin` as it is now is fine

Reply:
The benefit or returning DecCoin is it simplifies test cases. Without adding this change, we need to determine the dust prior to claiming incentives, then add that dust to the amount that the method says that it forfeits. In reality, this is the actual value it forfeits, so it makes the most sense imo to return this value. Otherwise we are doing hacky work arounds to determine what the value we actually forfeited which feels more odd

Reply:
Maybe we can modify the user query to return a coin object instead, but this feels the most natural since we need to know this value to determine if incentives were properly forfeited

Reply:
Do we want this API-breaking change this late in the game? What do you think about waiting until `v16.x` branch is cut and then merge this to `main` so that it goes into v17?

My only concern is causing unnecessary friction with integrations for something that is mostly useful for test convenience

Reply:
I am for merging this PR, it feels like the API breakage here can't be dangerous + this is the correct value anyways that we should have been returning

Reply:
I think it makes sense for the API to return the actual amount forfeited